run.sh

```diff
@@ -565,24 +562,25 @@ acraengdemo_launch_project_cockroachdb() {
     acraengdemo_run_compose
 }
 
-acraengdemo_launch_project_python-mysql() {
+acraengdemo_launch_project_python-mysql-postgresql() {
     COSSACKLABS_ACRA_VCS_URL=${COSSACKLABS_ACRA_VCS_URL:-'https://github.com/cossacklabs/acra'}
-    COSSACKLABS_ACRA_VCS_BRANCH=${COSSACKLABS_ACRA_VCS_BRANCH:-0.93.0}
+    # using commit instead of version/tag to use last commit with specific SqlAlchemy from examples/python
```

Comment:
let's leave here link to the specific commit/PR to simplify finding the reason and changes